packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -368,6 +383,11 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             commandId: CommonCommands.AUTO_SAVE.id
         });
 
+        registry.registerMenuAction(CommonMenus.FILE_CLOSE, {
+            commandId: CommonCommands.CLOSE_EDITOR.id,
+            label: 'Close Editor'
```

Comment:
here is label mentions Editor

Reply:
I added this menu action to be in line with VS Code. IMO a Label like "Close Main Area Tab" looks too complicated for the main menu, I'd rather drop the menu action then.

Reply:
Interesting that VS Code has `Close Window` and `Close Editor` for the same keybinding. Yes I would be fine to drop it.